config/initializers/assets.rb

```diff
@@ -18,6 +18,7 @@
                   host_edit
                   host_edit_interfaces
                   hosts
+                  jquery.ui.custom_spinners
```

Comment:
are you sure? I thought we have to wait for https://github.com/theforeman/foreman/pull/3811